storage/Filesystems/BlkFilesystem.h

```diff
@@ -46,13 +46,13 @@ namespace storage
 	static std::vector<BlkFilesystem*> get_all(Devicegraph* devicegraph);
 	static std::vector<const BlkFilesystem*> get_all(const Devicegraph* devicegraph);
 
-	virtual bool supports_label() const = 0;
-	virtual unsigned int max_labelsize() const = 0;
+	bool supports_label() const;
```

Comment:
Is it really a good idea to move the pure virtual functions from the public class to the impl class?
If you try to call such a pure virtual function on the base class, the linker will complain. But what happens when this is now in the impl class? Will the linker still detect the problem?